app/services/tool_registry.py

```diff
@@ -20,3 +20,31 @@ class ToolFile(BaseModel):
     filePath: Optional[str] = None
     url: str
     filename: Optional[str] = None
+    filetype: str
+
```

Comment:
LGTM

Reply:
This is great work, can this team also create documentation on the new loaders that they had created such as detailed information on the filters? Would love to see it so we can pass it to the front end team.

Reply:
Sure @mikhailocampo . We will create documentation on usage of the loaders and their parameters, processing and responses.